### DISCUSSION TITLE
Bad gateway for specific listing

### DISCUSSION THREAD

**Discussion by assaflei** [etsy/open-api#815] [Q&A]
Using the API [getListingsByListingIds](https://developer.etsy.com/documentation/reference/#operation/getListingsByListingIds) for specific listing returns 502 bad gateway error

It happens for a specific listing, can you assist with diagnose of this specific situation?
Due to customer data I rather share the details offline

Thanks,

**Comment by assaflei** [comment]
Anyone?

**Comment by baahar** [comment]
This happens to me if I add a file to a listing that was a physical listing with variation options. It becomes damaged beyond repair so that even if I open the listing to edit on Etsy there is an error message on top and no way to save changes, only preview. Hope this helps in some way.

**Comment by assaflei** [comment]
Thanks @baahar for your input, it a possible direction I would use to follow up on
I know that Etsy's dev team has an option to trace the api requests internally and share additional details
Hopefully someone from there could look at this as well..

**Comment by amitkumar-248** [comment]
@nateswart I'm also facing the same issue for the last 1 month. Plz help me out with what I need to do for those listings that are [corrupted.  ](https://github.com/etsy/open-api/discussions/815#discussioncomment-4706672) 
I'm attaching a curl request with a response header for the ref for one of my sellers. 

> Request post data 

`curl --location --globoff 'https://openapi.etsy.com/v3/application/shops/43159484/listings?shop_id=43159484&state=active&limit=1&includes[]=Inventory&offset=2' \
--header 'x-api-key: API_KEY' \
--header 'Authorization: Bearer ACCESS_TOKEN' \


> Response 

`{
    "error": "Bad Gateway"
}`

> Response header

`X-Etsy-Request-Uuid: EuSw_WWnn1O4YCCpfoSZxPW7c2f6`

> when I'm opening that listing_id  directly from Etsy, it's showing perfectly for the customers. attaching listing link below
[https://www.etsy.com/listing/1471450289/how-to-eliminate-self-limiting-beliefs](https://www.etsy.com/listing/1471450289/how-to-eliminate-self-limiting-beliefs)

Hopefully, this will help you in backtracing. 
waiting for a positive response. 
thanks
